lib/Chat/Changelog/Manager.php

```diff
@@ -144,6 +144,10 @@ public function getChangelogs(): array {
 			. $this->l->t('- Schedule a meeting into your calendar from within a conversation') . "\n"
 			. $this->l->t('- Search for messages of the current conversation directly in the right sidebar') . "\n"
 			. $this->l->t('- See more conversations on a first glance with the new compact list (enable in the Talk settings)'),
+			$this->l->t('## New in Talk %s', ['21.1']) . "\n"
+			. $this->l->t('- Event conversations now sync the title and description from the calendar and are hidden with a search filter until they are upcoming shortly') . "\n"
```

Comment:
```suggestion
			. $this->l->t('- Event conversations now sync the title and description from the calendar and are hidden with a search filter until they are close to the start date') . "\n"
```

Reply:
start date is either 'today' or 'tomorrow', maybe `close to the start time`?